internal/fnapi/fnapi.go

```diff
@@ -60,7 +60,7 @@ type Call[RequestT any] struct {
 	Method                 string
 	PreAuthenticateRequest func(*auth.UserAuth, *RequestT) error
 	OptionalAuth           bool // Don't fail if not authenticated.
```

Comment:
Now that telemetry and versions won't attach tokens, we can remove `OptionalAuth`